lib/membrane/rtp/metrics.ex

```diff
@@ -1,30 +1,35 @@
 defmodule Membrane.RTP.Metrics do
-  @moduledoc false
+  @moduledoc """
+  Defines list of metrics, that can be aggregated based on events from membrane_rtp_plugin.
+  """
 
+  @doc """
+  Returns list of metrics, that can be aggregated based on events from membrane_rtp_plugin.
+  """
   @spec metrics() :: [Telemetry.Metrics.t()]
   def metrics() do
     [
       Telemetry.Metrics.counter(
         "inbound-rtp.keyframe_request_sent",
-        event_name: [:sending_fir, :rtcp]
+        event_name: [Membrane.RTP, :RTCP, :fir, :sending]
       ),
       Telemetry.Metrics.counter(
         "inbound-rtp.packets",
-        event_name: [:packet_arrival, :rtp]
+        event_name: [Membrane.RTP, :RTP, :packet, :arrival]
```

Comment:
I'd prefix RTCP events only, or at least not use capital letters

```suggestion
        event_name: [Membrane.RTP, :rtp, :packet, :arrival]
```